src/init.cpp

```diff
@@ -1662,11 +1662,12 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
         static_cast<uint16_t>(args.GetIntArg("-port", Params().GetDefaultPort()));
 
     const auto BadPortWarning = [](const char* prefix, uint16_t port) {
-        return strprintf(_("%s request to listen on port %u. This port is considered \"bad\" and "
-                           "thus it is unlikely that any Bitcoin Core peers connect to it. See "
+        return strprintf(_("%s requests to listen on port %u. This port is considered \"bad\" and "
```

Comment:
`-bind` is more of an action than a noun, so I think the original here is correct

Reply:
Thanks. A commit [dropped](https://github.com/bitcoin/bitcoin/pull/26142#issuecomment-1260104579).